### PR TITLE
DDF-3543 Updates "Add Visual" tooltip

### PR DESCRIPTION
**Pull request by vinamartin**
#### What does this PR do?
Updates "Add Visual" tooltip to be slightly more descriptive.
I'm open to different wording here. 

#### Who is reviewing it? 
@emanns95 @ricklarsen @andrewkfiedler @Lambeaux @blen-desta @peterhuffer 

#### Choose 2 committers to review/merge the PR. 
@coyotesqrl 
@rzwiefel

#### How should this be tested? (List steps with links to updated documentation)
Go to Intrigue and open up a workspace.
Click the Question Mark icon in the upper right corner of the workspace.
Click on the Add Visual button and verify the tooltip is more descriptive. 

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: catalog/ui/catalog-ui-search/src/main/webapp/component/golden-layout/golden-layout.hbs [hunk 1]
```diff
@@ -17,7 +17,7 @@
 <div class="golden-layout-toolbar">
     <span></span>
     <div class="toolbar-actions">
-        <div class="to-add" title="Add visualization" data-help="Add visualiation">
+        <div class="to-add" title="Add visualization" data-help="Add a new visualization tab to the current view. Using a variety visualizations can help when looking at results.">
```
Comment (peterhuffer) [inline catalog/ui/catalog-ui-search/src/main/webapp/component/golden-layout/golden-layout.hbs]: ✏️ `... Using a variety of visualizations...`.

Comment (peterhuffer) [reply catalog/ui/catalog-ui-search/src/main/webapp/component/golden-layout/golden-layout.hbs]: May be clearer also noting also that visualizations are used to view Metacard results in different ways.

Comment (vinamartin) [reply catalog/ui/catalog-ui-search/src/main/webapp/component/golden-layout/golden-layout.hbs]: Good catch

**Comment by vinamartin** [comment]
build now

**Pull request merged**